core/Run.ml

```diff
@@ -634,12 +666,10 @@ let run_tests ~(mona : _ Mona.t) ~always_show_unchecked_output
       cont exit_code tests_with_status |> ignore;
       (* The continuation 'cont' should exit but otherwise we exit once
          it's done. *)
-      (* nosemgrep: forbid-exit *)
       exit exit_code)
   |> ignore;
   (* shouldn't be reached if 'bind' does what it's supposed to *)
-  (* nosemgrep: forbid-exit *)
```

Comment:
How were you able to remove the nosemgrep? Hmm maybe because now the code is in a submodule
so semgrep rules do not apply in submodules given our current file targeting in osemgrep?